docs/common/standards.rst

```diff
@@ -1,97 +1,135 @@
 .. include:: ../common/common_definitions.rst
 
+
 Technical References
 ++++++++++++++++++++
 
+This section details the technical references, grouped by scope, that are part of the current implementation profile or for its future milestones.
+This section includes international and national standards, draft specifications, architecture references and other relevant technical documents
+foundational for the interoperability and the compliance with industry standards.
+
+
+
+Wallet Paradigm Frameworks
+--------------------------
+
 .. list-table::
     :widths: 25 75
     :header-rows: 0
 
-    * - `OID-FED`_
-      - Hedberg, R., Jones, M.B., Solberg, A.Å., Bradley, J., De Marco, G., Dzhuvinov, V.,  "OpenID Federation 1.0", December 2024, Draft 41.
-    * - `OpenID4VCI`_
-      - Lodderstedt, T., Yasuda, K., Looker, T., "OpenID for Verifiable Credential Issuance", December 2024, Draft 15.
-    * - `SD-JWT-VC`_
-      - O. Terbu, D.Fett, B. Campbell, "SD-JWT-based Verifiable Credentials (SD-JWT VC)".
     * - `EIDAS-ARF`_
       - EUDI Wallet - Architecture and Reference Framework.
+
+
+Infrastructure of Trust
+-----------------------
+
+.. list-table::
+    :widths: 25 75
+    :header-rows: 0
+
+    * - :rfc:`5280`
+      - Cooper, D., Santesson, S., Farrell, S., Boeyen, S., Housley, R., and W. Polk, "Internet X.509 Public Key Infrastructure Certificate and Certificate Revocation List (CRL) Profile", RFC 5280, May 2008.
+    * - `OID-FED-WALLET`_
+      - De Marco, G., Hedberg, R., Jones, M.B., Bradley, J., Dzhuvinov, V., "OpenID Federation Wallet Architectures 1.0", October 2024, Draft 03.
+
+
+Digital Credential Data Format
+------------------------------
+
+.. list-table::
+    :widths: 25 75
+    :header-rows: 0
+
+    * - `SD-JWT-VC`_
+      - O. Terbu, D.Fett, B. Campbell, "SD-JWT-based Verifiable Credentials (SD-JWT VC)".
+    * - ISO18013-5
+      - ISO/IEC 18013-5 2020. Information technology — Personal identification — ISO-compliant driving license — Part 5: Mobile driving license (mDL) application.
+    * - ISO-IEC_7367
+      - ISO Compliant Mobile Vehicle Registration Certificates, Logical Data Structure
+
+
+Digital Credential Issuance
+---------------------------
+
+.. list-table::
+    :widths: 25 75
+    :header-rows: 0
+
+    * - `OpenID4VCI`_
+      - Lodderstedt, T., Yasuda, K., Looker, T., "OpenID for Verifiable Credential Issuance", December 2024, Draft 15.
+    * - `OPENID4VC-HAIP`_
+      - Lodderstedt, T., K. Yasuda, "OpenID4VC High Assurance Interoperability Profile", January 2025, Draft 1.
```

Comment:
We have no compliance on this profile. CC @fmarino-ipzs

Reply:
we are referring to this profile within our technical specification. This doesn't mean that we have full compliance. The same applies to IDA profile. I think we can keep this reference.